client/benchmark/src/main/java/org/elasticsearch/client/benchmark/BenchmarkRunner.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.client.benchmark;
+
+import org.elasticsearch.client.benchmark.metrics.Metrics;
+import org.elasticsearch.client.benchmark.metrics.MetricsCalculator;
+import org.elasticsearch.client.benchmark.metrics.Sample;
+import org.elasticsearch.client.benchmark.metrics.SampleRecorder;
+import org.elasticsearch.common.SuppressForbidden;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+
+public final class BenchmarkRunner {
+    private final int warmupIterations;
+    private final int iterations;
+    private final BenchmarkTask task;
+
+    public BenchmarkRunner(int warmupIterations, int iterations, BenchmarkTask task) {
+        this(warmupIterations, iterations, 1, task);
+    }
+
+    public BenchmarkRunner(int warmupIterations, int iterations, int operationsPerIteration, BenchmarkTask task) {
```

Comment:
Right. This was a leftover. I've removed this constructor now. Thanks for spotting it.